### ISSUE TITLE
[feature-request] Extend config

### ISSUE THREAD

**Issue by sushruth** [bpierre/esbuild-config#4]
Would it be possible to add an "extend" property? I am trying to set up a monorepo with esbuild and this would make most common configuration very easy.

**Comment by bpierre** [comment]
Fantastic idea! I’ll add it :+1:

**Comment by sushruth** [comment]
@bpierre just wondering if this is considered as an accepted planned feature for this. Thank you again.

**Comment by bpierre** [comment]
Definitely! I haven’t started yet, but it is the next thing I plan to add :slightly_smiling_face:
